engine/src/test/scala/cromwell/services/CromwellServicesTest.scala

```diff
@@ -0,0 +1,64 @@
+package cromwell.services
+
+import akka.actor.ActorRef
+import akka.util.Timeout
+import com.typesafe.config.{Config, ConfigFactory}
+import cromwell.CromwellTestkitSpec.TestWorkflowManagerSystem
+import cromwell.backend.{BackendJobDescriptor, BackendJobDescriptorKey, BackendWorkflowDescriptor}
+import cromwell.core.{WorkflowId, WorkflowOptions}
+import cromwell.engine.WorkflowSourceFiles
+import cromwell.engine.backend.local.LocalBackend
+import cromwell.engine.backend.{WorkflowDescriptor, WorkflowDescriptorBuilder}
+import cromwell.engine.db.DataAccess
+import cromwell.engine.db.slick.SlickDataAccess
+import org.scalatest.concurrent.ScalaFutures
+import org.scalatest.time.{Millis, Seconds, Span}
+import org.scalatest.{BeforeAndAfterAll, FlatSpec, Matchers}
+import spray.json.JsObject
+
+import scala.concurrent.duration._
+
+trait CromwellServicesTest extends FlatSpec with Matchers with BeforeAndAfterAll with WorkflowDescriptorBuilder with ScalaFutures {
```

Comment:
Previous comment mentioned that our current convention for test code is `*Spec` not `*Test`.